platformio.ini

```diff
@@ -12,3 +12,4 @@
 platform = teensy
 board = teensy41
 framework = arduino
+; build_flag = -I path   
```

Comment:
Is this necessary?

Reply:
Should be removed. I used it as an example of how to include other files using platformio

Reply:
if anybody needs an example on how to include other libraries and stuff on platformio


[env:teensy41]
platform = teensy
board = teensy41
framework = arduino
build_flags = -L ./lib/micro_ros_arduino/src/imxrt1062/fpv5-d16-hard/ 
              -l libmicroros
              -I /opt/ros/galactic/install/